File: Programas em Java/04 - Laço For com Vetores/Script4.java
/*
 * Escreva um algoritmo que leia e mostre um vetor de 20 elementos 
 * inteiros, em seguida, mostre a soma de todos os elementos.
 */

package geek;

import java.util.Scanner;

public class Script4 {

	public static void main(String[] args) {
		//variaveis
		int[] vetor = new int[20];
		int soma = 0;
		Scanner teclado = new Scanner(System.in);
		
		//entrada
		for (int i = 0; i < 20; i++) {
			System.out.printf("Informe o valor para o vetor %d/20\n", (i+1));
			vetor[i] = teclado.nextInt();
			soma = soma + vetor[i];
		}
		System.out.printf("A soma é %d", soma);
		teclado.close();

	}

}

File: Programas em Python/04 - Vetores/Script4.py
# Escreva um algoritmo que leia e mostre um vetor de 20 elementos interios, 
# em seguida, mostre a soma de todos os elementos.

#variáveis
vetor = []
#soma = 0
#entradas
for n in range(0, 20):
    num = int(input("Informe {0}/20 valor para o vetor: ".format(n+1)))
    vetor.append(num)
    #soma = soma + num
print("A soma do vetor é {0}".format(sum(vetor)))
